scripts/validate_docstrings.py

```diff
@@ -15,40 +15,28 @@
 """
 from __future__ import annotations
 
-import argparse
+from argparse import ArgumentParser
 import doctest
-import glob
-import importlib
-import json
-import os
-import subprocess
+from importlib import import_module
```

Comment:
IMO I prefer the older import style as opposed to this way since it makes things more explicit where these functions & objects come from in the script